lib/util/hud_code_gen.rb

```diff
@@ -141,4 +142,108 @@ module Types::HmisSchema::Enums::Hud
 
     [map_name, lookup_fn_name]
   end
+
+  def generate_hud_list_json(year, excel_file_path)
+    # Parse the existing JSON data. This will be the base from which we start.
+    json_file_path = "lib/data/#{year}_hud_lists.json"
+    json_data = JSON.parse(File.read(json_file_path))
+
+    # TODO: is there a way to better skip these?
+    ignore_codes = ['List', '*', '(see note)']
```

Comment:
Seems reasonable to me.
```suggestion
    # Some codes are irrelevant to this transformation, we'll ignore them based on string comparison
    ignore_codes = ['List', '*', '(see note)']
```